webrtc/videoroom_advanced/lib/videoroom/room.ex

```diff
@@ -29,15 +36,25 @@ defmodule Videoroom.Room do
       id: room_id
     ]
 
+    turn_cert_file =
+      case Application.fetch_env(:membrane_videoroom_demo, :integrated_turn_certfile) do
+        {:ok, val} -> val
+        :error -> nil
+      end
+
     network_options = [
       stun_servers: Application.fetch_env!(:membrane_videoroom_demo, :stun_servers),
       turn_servers: Application.fetch_env!(:membrane_videoroom_demo, :turn_servers),
       use_integrated_turn: Application.fetch_env!(:membrane_videoroom_demo, :use_integrated_turn),
       integrated_turn_options: [
         ip: turn_ip,
         mock_ip: turn_mock_ip,
-        ports_range: Application.fetch_env!(:membrane_videoroom_demo, :integrated_turn_port_range)
+        ports_range:
+          Application.fetch_env!(:membrane_videoroom_demo, :integrated_turn_port_range),
+        cert_file: turn_cert_file
       ],
+      integrated_turn_domain:
+        Application.fetch_env!(:membrane_videoroom_demo, :integrated_turn_domain),
```

Comment:
I would put this in `integrated_turn_options`

Reply:
`integrated_turn_options` is part of `Membrane.ICE.Endpoint` API. `integrated_turn_domain` is not used by `Membrane.ICE.Endpoint` or any of it's dependencies, so I decided to not put it there